moz-webgpu-cts/src/shared.rs

```diff
@@ -477,7 +488,8 @@ impl<'a> TestPath<'a> {
         })
     }
 
-    pub fn from_fx_metadata_test(
+    pub fn from_metadata_test(
+        browser: Browser,
```

Comment:
issue: We should force calling code to specify what `Browser` they're expecting. This can avoid some issues we otherwise might have with this code falling back to another browser's parent file prefix.

Pushed `fixup!`s 0c54b62, 9ba6deb.